test/assets/group_create_async.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<tsResponse xmlns="http://tableau.com/api"
+            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xsi:schemaLocation="http://tableau.com/api http://tableau.com/api/ts-api-2.3.xsd">
+    <job id="job-id" mode="Asynchronous" type="GroupImport"
```

Comment:
Yup, that's for a future test when I implement import() (see comment above about AD import)